package.json

```diff
@@ -151,6 +151,7 @@
     "**/jake/async": "^2.6.4",
     "**/json5": "^2.2.3",
     "**/nth-check": "^2.0.1",
+    "**/pkg/prebuild-install": "7.1.1",
```

Comment:
Fixes. 
![image](https://user-images.githubusercontent.com/5849952/220176089-3918ab98-b5ef-4a25-9d8b-21009d961dfc.png)

We might need to keep it until https://github.com/vercel/pkg/commit/e3ac4902e9cb023f0ac2a596e21ff9166d2e268c ships.